third-readme.md

```diff
@@ -0,0 +1,28 @@
+# Get a head start
```

Comment:
Let's go with _Development preparations_ as a header and name the file `dev-prep.md`.